### PR TITLE
Fix memory leaks

### PR DESCRIPTION
**Pull request by rbuj**

### CODE REVIEW

File: src/fr-command-tar.c [hunk 1]
```diff
@@ -140,8 +140,13 @@ process_line (char     *line,
 	}
 	g_free (name);
 	name = g_filename_from_utf8 (fdata->original_path, -1, NULL, NULL, NULL);
-	if (name)
-		fdata->original_path = name;
+	if (name) {
+		if (fdata->free_original_path)
+			g_free (fdata->original_path);
```
Comment (mbkma) [inline src/fr-command-tar.c:145]: redundant null check

**Comment by mbkma** [comment]
other than that its fine

File: src/fr-command-tar.c (hunk 1 above)
Comment (rbuj) [reply src/fr-command-tar.c]: done

**Review by mbkma — APPROVED** [review]
LGTM

File: src/fr-command-tar.c (hunk 1 above)
Comment (rbuj) [reply src/fr-command-tar.c:145]: @mbkma `fdata->original_path` is not NULL and points to `fdata->full_path`.

**Comment by raveit65** [comment]
@rbuj 
Can you please trigger a deployment at travis?

**Review by raveit65 — APPROVED** [review]
LGTM

**Pull request merged**